### DISCUSSION TITLE
stato tv (con ping)

### DISCUSSION THREAD

**Discussion by RtoRoc** [indomus/forum#163] [HUB: Home Assistant]
buon giorno.
sto cercando di dedurre lo stato del mio tv utilizzando il ping. quindi mi sono creato un binary_sensor che mi restituisce lo stato connesso/disconnesso dopo di che mi sono fatto un template acceso/spento. inizialmente sembrava funzionare perfettamente. poi pero mi sono reso conto quando spengo la tv lo stato mi passa a spento  ma dopo circa 1 minuto mi mi passa ad acceso sebbene la tv sia spenta.  svete consigli? suggerimenti? soluzioni? 
```
binary_sensor:
  - platform: ping
    host: 192.168.1.145
    name: "Sony Bravia"
    count: 2
    scan_interval: 5
    
sensor:
  - platform: template
    sensors:
      sony_bravia:
        friendly_name: "Sony Bravia"
        icon_template: 'mdi:television'
        value_template: >-
          {% if states('binary_sensor.sony_bravia') == "on" %}
            Acceso
          {% else %}
            Spento
          {% endif %}```

**Comment by 50494554524F** [comment]
ping mica ti dice se è acceso o spento, solo se è visibile nella rete locale

vedi qui se può fare al caso
https://www.home-assistant.io/integrations/braviatv/

**Reply by RtoRoc** [reply]
grazie per la risposta pietro... si lo so che non ti da acceso e spento in fatti ho creato un template che mi dia acceso e spento in base a se sia connesso o meno.... comuqneu dopo una giornata di cristi tirati in terra ho monitorato la rituazione ed ho notato e e il tv che ha un comportamento anomalo in fatti quando lo spengo  resta presente in rete per piu o meno un oretta dopo di che effettivamente non viene piu pingato e quindi mi risulta spento... na bella rottura insomma

**Reply by RtoRoc** [reply]
purtoroppo quel integrazione per il mio modello di tv non va bene, gia l'avevo guardata e provata

**Comment by Scialla** [comment]
prenditi una blitzwolf sph13 zigbee e ti rilevi lo stato dall'assorbimento. Non vedo altre strade se l'integrazione non va, io ho fatto così

**Reply by MarioBomb** [reply]
Io uso uno Shelly PlugS per lo stesso motivo.

**Reply by Jamiro75** [reply]
Se la USB del televisore si accende con esso può bastare anche un ESP 8266 con tasmota/esphome o altro e rilevi quando è connesso senza altro codice, 
Stupido ma funziona e si può usare anche su altri device che si accendono ed alimentano USB

**Reply by RtoRoc** [reply]
> Se la USB del televisore si accende con esso può bastare anche un ESP 8266 con tasmota/esphome o altro e rilevi quando è connesso senza altro codice,
> Stupido ma funziona e si può usare anche su altri device che si accendono ed alimentano USB

ma tu sei un grande..... non ci avevo pensato alla usb..... ho un sonoff di quelli a 5v. buttato dentro un cassetto, posso utilizzarlo al posto del esp(che attualmente non ho) bella  idea. grazie
